mypy.ini

```diff
@@ -1,5 +1,5 @@
 [mypy]
-python_version = 3.9
+python_version = 3.10
```

Comment:
Typically we've used mypy with oldest version, so that if someone uses tooling which depends on types in the oldest supported version it works out. Due to that, as we still plan to support 3.9, this doesn't feel like good idea (might invalidate some of the changes too).

Reply:
Right. Changed it while debugging some mypy failures, reverted now.